### PR TITLE
Feature/WAL-679

### PR DESCRIPTION
**Pull request by BtencateSphereon**

### CODE REVIEW

File: src/styles/colors.ts [hunk 1]
```diff
@@ -1,134 +1,105 @@
...
-  secondaryLight: '#E3E3FF',
-};
-
-type Alerts = 'primaryLight' | 'secondaryLight';
-
-export const alerts: Record<Alerts, string> = {
-  primaryLight: '#FBFBFB',
-  secondaryLight: '#EBEBEB',
-};
+// type Background = 'primaryDark' | 'secondaryDark' | 'primaryLight' | 'secondaryLight';
```
Comment (nklomp) [inline src/styles/colors.ts:3]: Remove commented code here and below

**Review by nklomp — CHANGES_REQUESTED** [review]
Wait for ui components build and 1 remark about some commented code

File: src/styles/colors.ts (hunk 1 above)
Comment (BtencateSphereon) [reply src/styles/colors.ts]: removed

**Review by nklomp — APPROVED** [review]
Lgtm

**Pull request merged**